Armadillo/src/main/java/com/scribd/armadillo/playback/MediaSessionCallback.kt

```diff
@@ -138,10 +134,11 @@ internal class MediaSessionCallback(private val onMediaSessionEventListener: OnM
             onStop()
         }
 
-        @Suppress("UNCHECKED_CAST") val initialOffset = extras.getSerializable(Constants.Keys.KEY_INITIAL_OFFSET) as Milliseconds
+        val config = extras.getSerializable(Constants.Keys.KEY_ARMADILLO_CONFIG) as ArmadilloConfiguration
```

Comment:
Is there any way this could be missing?

Reply:
I don't believe so, but better to err on the safe side. In case it's missing from the bundle, it'll come back as null, so I'll provide a default value for the config.

Reply:
Yeah, depends on what direction you wanna go with it

If there is no way it could be missing, can probably go with what you had before and we'll just get hit with a NPE or typecast exception we can investigate if there's a problem. Keeps the code simpler and confidently shows clear intent of how it's working/what _should_ be happening

Having a fallback is safe, but it also obscures when something is wrong, so usually good to pair with logging an event to our dashboard. Not sure if we do that in Armadillo without it throwing an error that might stop the player

Reply:
Applying that to the config, I feel like it makes sense that I should always be expecting the configuration for the audio playable so I'll go with the former. And good point about fallback possibly obscuring errors.